### ISSUE TITLE
Determine if the mean gains across subjects from perturbed and unperturbed are significantly different. 

### ISSUE THREAD

**Issue by moorepants** [csu-hmc/gait-control-direct-id-paper#54]
I think this can be done with a two-sample T test.

**Comment by tvdbogert** [comment]
Yes. I think what you refer to is also known as "paired T test".  If you have more than two cases to compare, it becomes ANOVA.

If you have 240 gain value comparisons, the type 1 error is an issue.  If you use the p<0.05 criterion, 5% of the comparisons will show up as significantly different even if the data consists of random numbers.  There's a Bonferroni correction method that says to use p < 0.05/Ncomparisons but that seems too strict.

Best may be to put this in a 2-way ANOVA, with these three discrete factors: perturbed (2 levels, yes or no), time in gait cycle (20 levels).  Then you only have 6 comparisons.  Or even a 3-way ANOVA where the 3rd factor is which gain you are looking at.

I am not a statistics expert, but it sounds like a good idea to make the presentation of results more scientific by statistical tests instead of just saying "looks the same".

**Comment by moorepants** [comment]
> I think what you refer to is also known as "paired T test".

Yes.

> If you have 240 gain value comparisons, the type 1 error is an issue. If you use the p<0.05 criterion, 5% of the comparisons will show up as significantly different even if the data consists of random numbers. There's a Bonferroni correction method that says to use p < 0.05/Ncomparisons but that seems too strict.

I'm not sure that I need to compare each of the 240 gain values to all the others. Why do you think so?

> Best may be to put this in a 2-way ANOVA, with these three discrete factors: perturbed (2 levels, yes or no), time in gait cycle (20 levels). Then you only have 6 comparisons. Or even a 3-way ANOVA where the 3rd factor is which gain you are looking at.

I could certainly do some larger multi-variate tests but I was trying to think of the simplest stats that I could run to bolster any claims we make.

> I am not a statistics expert, but it sounds like a good idea to make the presentation of results more scientific by statistical tests instead of just saying "looks the same".

I have a poor understanding of all of this too.

At a minimum I have these things to factor for the joint isolated results:

Dependent variables (240 values):
- 6 mass normalized gains from the left side and 6 from the right side 
- 20 gait phase points

Independent variables:
- 11 subjects
- 3 walking speeds (one of each speed for each subject)
- perturbed/unperturbed
- gender (probably not gonna say anything about this)

The statements I'm hoping to claim are:
1. The gains are similar among subjects.
2. Which gains are potentially zero and which are not. [Issue #51]
3. The gains are similar for the right and left sides. [Issue #53]
4. The gains are similar from unperturbed and perturbed walking. [Issue #54]
5. A single gain changes significantly through its schedule. [Issue #52]
6. Whether the gains change with respect to speed.

When I collect the scheduled gains at one speed, one perturbation type, for all subjects, I have 240 gain values x 11 subjects/trials. If I look at the distribution of each scheduled gain value across the 11 subjects they do not seem to be normally distributed (checking a quantile-quantile plot).

I thought that this is bare minimum I need to do for the above six items:
1. Find the mean and standard deviation across subjects and state whether the std is large or small for each gain.
2. Run a one sample t test compare each gain distribution across subjects to 0. This should show which gains are not zero.
3. Run a paired t test for each gain using the right and left distributions of 11 gain values. This will tell me if each gain from the right is significantly different from the one on the left.
4. Same as 3 but comparing each gain subject distribution from unperturbed and perturbed.
5. For each gain run a one way ANOVA among the 20 scheduled values and post hoc analyses to see which ones are different.
6. For each scheduled gain value run a one way ANOVA with respect to the three speeds.

Do you think this is correct way to approach it? Or do I need to do something more complicated to deal with all the variable types?
